app/src/main/res/values-nl/strings.xml

```diff
@@ -111,28 +111,28 @@
 
     <!-- View types -->
     <string name="change_view_type">Weergave</string>
-    <string name="grid">Rooster</string>
+    <string name="grid">Raster</string>
     <string name="list">Lijst</string>
 
     <!-- Settings -->
     <string name="autoplay_videos">Video\'s automatisch afspelen</string>
-    <string name="toggle_filename">Bestandsnamen tonen</string>
+    <string name="toggle_filename">Bestandsnamen weergeven</string>
     <string name="loop_videos">Video\'s herhalen</string>
-    <string name="animate_gifs">GIF-bestanden afspelen in overzicht</string>
+    <string name="animate_gifs">GIF\'s afspelen in overzicht</string>
     <string name="max_brightness">Maximale helderheid in volledige weergave</string>
-    <string name="crop_thumbnails">Voorbeelden vierkant bijsnijden</string>
-    <string name="screen_rotation_by">Media in volledige weergave roteren volgens</string>
+    <string name="crop_thumbnails">Miniatuurvoorbeelden bijsnijden</string>
+    <string name="screen_rotation_by">Media in volledige weergave draaien volgens</string>
     <string name="screen_rotation_system_setting">Systeeminstelling</string>
-    <string name="screen_rotation_device_rotation">Rotatie van apparaat</string>
+    <string name="screen_rotation_device_rotation">Draaiing van apparaat</string>
```

Comment:
You changed it to "oriëntatie" earlier...

Reply:
Good catch